cmd/lakectl/cmd/root.go

```diff
@@ -87,7 +107,64 @@ var rootCmd = &cobra.Command{
 			DieFmt("error unmarshal configuration: %v", err)
 		}
 	},
-	Version: version.Version,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		if !MustBool(cmd.Flags().GetBool("version")) {
+			if err := cmd.Help(); err != nil {
+				WriteIfVerbose("failed showing help {{ . }}", err)
+			}
+			return
+		}
+
+		verbose := MustBool(cmd.Flags().GetBool("verbose"))
+
+		info := versionInfo{LakectlVersion: version.Version}
+
+		// get lakeFS server version
+
+		client := getClient()
+		resp, err := client.GetLakeFSVersionWithResponse(context.Background())
```

Comment:
In `lakectl` we don't use timeout for the client, and I'm not sure we should since it's a CLI, anyway -  I prefer not to diverge  from that logic.
 We are covered in terms of Github source, there is a timeout there.